File: app/src/main/java/io/github/lee0701/heonot/inputmethod/modules/hardkeyboard/HardKeyboard.java
package io.github.lee0701.heonot.inputmethod.modules.hardkeyboard;

import io.github.lee0701.heonot.inputmethod.event.HardKeyEvent;
import io.github.lee0701.heonot.inputmethod.modules.InputMethodModule;

public abstract class HardKeyboard extends InputMethodModule {

	public abstract void input(HardKeyEvent event);

}

File: heonot-core/src/main/java/io/github/lee0701/heonot/inputmethod/modules/hardkeyboard/HardKeyboard.kt
package io.github.lee0701.heonot.inputmethod.modules.hardkeyboard

import io.github.lee0701.heonot.inputmethod.event.CommitCharEvent
import io.github.lee0701.heonot.inputmethod.event.HardKeyEvent
import io.github.lee0701.heonot.inputmethod.modules.InputMethodModule
import org.greenrobot.eventbus.EventBus

abstract class HardKeyboard : InputMethodModule() {
    abstract fun input(event: HardKeyEvent)
}
